packages/devtools_app/pubspec.yaml

```diff
@@ -61,7 +61,7 @@ dependencies:
   vm_service_protos: ^1.0.0
   # TODO https://github.com/dart-lang/sdk/issues/52853 - unpin this version
   vm_snapshot_analysis: ^0.7.6
-  web: ^0.4.1
+  web: ^0.5.0
```

Comment:
should these versions be 0.5.1?

Reply:
I tend to just specify the version required (here, `0.5.0` for the new API) rather than the latest bug fix version. Happy to be more specific here if desired.